bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/testcases/DigitalInvoiceScreenTests.kt

```diff
@@ -0,0 +1,199 @@
+        captureScreen.clickFilesButton()
+        captureScreen.clickFiles()
+        pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.displayDigitalInvoiceTextOnOnboardingScreen()
+        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+    }
+
+    @Test
+    fun test2_disableToggleSwitchToRemoveItemFromList() {
+        mainScreen.clickPhotoPaymentButton()
+        onboardingScreen.clickSkipButton()
+        captureScreen.clickFilesButton()
+        captureScreen.clickFiles()
+        pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.assertDigitalInvoiceText()
+        digitalInvoiceScreen.clickArticleSwitch()
+        val isItemEnabled =  digitalInvoiceScreen.checkItemIsDisabledFromDigitalScreen()
+        assertEquals(false, isItemEnabled)
+    }
+
+    @Test
+    fun test3_returnReasonDisplaysWhenToggleSwitchIsDisabled() {
+        mainScreen.clickSettingButton()
+        configurationScreen.scrollToUICustomizationText()
+        configurationScreen.clickReturnReasonsDialogToEnable()
+        pressBack()
+        mainScreen.clickPhotoPaymentButton()
+        onboardingScreen.clickSkipButton()
+        captureScreen.clickFilesButton()
+        captureScreen.clickFiles()
+        pdfUploader.uploadPdfFromFiles("Testrechnung-RA-1.pdf")
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.displayGetStartedButtonOnOnboardingScreen()
+        digitalInvoiceScreen.clickGetStartedButtonOnOnboardingScreen()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.assertDigitalInvoiceText()
+        digitalInvoiceScreen.clickArticleSwitch()
+        digitalInvoiceScreen.checkForReturnReasonsList()
+    }
+
+    @Test
+    fun test4_verifyCountOnReturnReasonsList() {
+        test3_returnReasonDisplaysWhenToggleSwitchIsDisabled()
+        digitalInvoiceScreen.checkItemCountOnReturnReasonsList()
+    }
+
+    @Test
+    fun test5_clickOnItemFromReturnReasonsList() {
+        test3_returnReasonDisplaysWhenToggleSwitchIsDisabled()
+        digitalInvoiceScreen.clickItemOnReturnReasonsList()
+        idlingResource.waitForIdle()
+        val isItemEnabled = digitalInvoiceScreen.checkItemIsDisabledFromDigitalScreen()
+        assertEquals(false, isItemEnabled)
+    }
+
+    @Test
+    fun test6_enableToggleSwitchToAddItemBackToList() {
+        test5_clickOnItemFromReturnReasonsList()
+        idlingResource.waitForIdle()
+        digitalInvoiceScreen.assertDigitalInvoiceText()
+        digitalInvoiceScreen.clickArticleSwitch()
+        val isItemDisabled =digitalInvoiceScreen.checkItemIsEnabledFromDigitalScreen()
+        assertEquals(true, isItemDisabled)
```

Comment:
[Clarification] From the name of the test, I understand that an item should be `enabled` after clicking on a toggle switch....however the assertion checks if the item is disabled (`isItemDisabled` is equal to `true`). 

Could we update either the title or the assertion to be in sync with what the goal of the test case is?

Reply:
yes, thank you for clarifying it! ❤️